### PR TITLE
timeline

### PR DESCRIPTION
**Pull request by flexsurfer**
timeline

### CODE REVIEW

File: protocol/messenger.go [hunk 1]
```diff
@@ -502,7 +502,7 @@ func (m *Messenger) Init() error {
 			continue
 		}
 		switch chat.ChatType {
-		case ChatTypePublic, ChatTypeProfile:
+		case ChatTypePublic, ChatTypeProfile, ChatTypeTimeline:
```
Comment (cammellos) [inline protocol/messenger.go]: This is used to create filters, do you need a filter for a timeline chat?

File: protocol/messenger.go [hunk 2]
```diff
@@ -3800,7 +3815,7 @@ func (m *Messenger) encodeChatEntity(chat *Chat, message common.ChatEntity) ([]b
 		if err != nil {
 			return nil, err
 		}
-	case ChatTypePublic, ChatTypeProfile:
+	case ChatTypePublic, ChatTypeProfile, ChatTypeTimeline:
```
Comment (cammellos) [inline protocol/messenger.go]: Same here, do you ever save messages for a timeline chat?

File: protocol/messenger.go (hunk 1 above)
Comment (flexsurfer) [reply protocol/messenger.go]: no thanks

File: protocol/messenger.go (hunk 2 above)
Comment (flexsurfer) [reply protocol/messenger.go]: no thanks

**Comment by flexsurfer** [comment]
thanks @cammellos fixed

**Review by cammellos — APPROVED** [review]

**Comment by cammellos** [comment]
@flexsurfer looks good to me, there's some linting issues

**Pull request merged**